src/components/image-config/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement, ReactNode } from 'react'
+import getValidatedImgChild from './get-validated-image-child'
+import MdxImg from '../mdx-img'
+import s from './image-config.module.css'
+
+/**
+ * Configure extra settings for images via MDX.
+ * Designed to wrap a single markdown image.
+ */
+export default function ImageConfig({
+  /** Hide borders (temporarily configurable while we remove baked-in borders from existing images) */
+  hideBorder = false,
```

Comment:
What's even more confusing to me about this prop RN (besides it being in a perma deprecated state) is that is basically has no effect on the actual rendering of the component — it always shows a border. I wonder if there is a way to just not define it as an option at all, but make it so if its passed the component doesn't break?